runtime-modules/bounty/src/benchmarking.rs

```diff
@@ -843,119 +1075,665 @@ benchmarks! {
+        };
+
+        Bounty::<T>::create_bounty(RawOrigin::Root.into(), params, Vec::new()).unwrap();
+
+        let bounty_id: T::BountyId = Bounty::<T>::bounty_count().into();
+
+    }: switch_oracle (
+        RawOrigin::Signed(current_oracle_account_id),
+        new_oracle.clone(),
+        bounty_id)
+    verify {
+        let bounty_id: T::BountyId = 1u32.into();
+
+        assert!(Bounties::<T>::contains_key(bounty_id));
+        assert_last_event::<T>(
+            Event::<T>::BountyOracleSwitched(
+                bounty_id,
+                BountyActor::Member(current_oracle_member_id),
+                oracle,
+                new_oracle).into());
+    }
+
+    end_working_period{
+        let cherry: BalanceOf<T> = 100u32.into();
+        let oracle_reward: BalanceOf<T> = 100u32.into();
         let funding_amount: BalanceOf<T> = 100u32.into();
-        let work_period: T::BlockNumber = One::one();
         let (oracle_account_id, oracle_member_id) = member_funded_account::<T>("oracle", 1);
         let oracle = BountyActor::Member(oracle_member_id);
         let stake: BalanceOf<T> = 100u32.into();
         let creator = BountyActor::Council;
 
         let params = BountyCreationParameters::<T> {
-            work_period,
-            judging_period: One::one(),
-            creator: creator.clone(),
+            creator,
             cherry,
+            oracle_reward,
             funding_type: FundingType::Perpetual{ target: funding_amount },
             oracle: oracle.clone(),
             entrant_stake: stake,
             ..Default::default()
         };
 
-        let bounty_id = create_max_funded_bounty::<T>(params);
+        let bounty_id = create_funded_bounty::<T>(params);
 
         let (work_account_id, work_member_id) = member_funded_account::<T>("work entrants", 0);
-
+        let work_description = b"work_description".to_vec();
         Bounty::<T>::announce_work_entry(
             RawOrigin::Signed(work_account_id.clone()).into(),
             work_member_id,
             bounty_id,
             work_account_id.clone(),
+            work_description
+        )
+        .unwrap();
+
+        let entry_id: T::EntryId = Bounty::<T>::entry_count().into();
+
+        let work_data = b"work_data".to_vec();
+        Bounty::<T>::submit_work(
+            RawOrigin::Signed(work_account_id).into(),
+            work_member_id,
+            bounty_id,
+            entry_id,
+            work_data,
+        )
+        .unwrap();
+
+    }: end_working_period(RawOrigin::Signed(oracle_account_id.clone()), bounty_id)
+    verify {
+        let bounty_id: T::BountyId = 1u32.into();
+
+        assert!(Bounties::<T>::contains_key(bounty_id));
+        assert_eq!(Bounty::<T>::bounty_count(), 1); // Bounty counter was updated.
+        assert_last_event::<T>(
+            Event::<T>::WorkSubmissionPeriodEnded(
+                bounty_id,
+                oracle).into());
+    }
+    withdraw_entrant_stake{
```

Comment:
Please, add lines between benchmarks.

Reply:
But `cargo clippy --package pallet-bounty --features runtime-benchmarks` will run all the dependencies, I would have to fix all other pallets like membership etc. 
I used `cargo clippy -p pallet-bounty --all-features -- --no-deps` instead,
Should I fix the other pallets?
Is the 80 length rule, applied only in benchmarking? Because there are situations where `cargo fmt` uses more than 80 col and is not a macro